src/main/rx/xbus.c

```diff
@@ -158,9 +181,68 @@ static void xBusUnpackFrame(void)
 
 }
 
+static void xBusUnpackRJ01Frame(void)
+{
+    // Calculate the CRC of the incoming frame
+    uint16_t crc = 0;
+    uint16_t inCrc = 0;
+    uint8_t i = 0;
+    uint16_t value;
+    uint8_t frameAddr;
+
+    // When using the Align RJ01 receiver with 
+    // a MODE B setting in the radio (XG14 tested)
+    // the MODE_B -frame is packed within some
+    // at the moment unknown bytes before and after:
+    // 0xA1 __ __ 0xA1 12*(High + Low) CRC1 CRC2 + __ __ __
```

Comment:
Best approach would IMO be to parse outer frame in MODE_B_RJ01 decoder, check that wrapper content is consistent (0x1e is probably packet length, 0x06 may be payload type) and then pass it to xBusUnpackModeBFrame. That way code duplication will be reduced and it will not break when receiver sends packet with other payload ...

If you post saleae save file, I can reverse-engineer  CRC used for outer packets ...